packages/theme-chalk/src/table.scss

```diff
@@ -634,4 +634,33 @@
       margin-right: 8px;
     }
   }
+
+  @include b(table) {
+    &.#{$namespace}-table--border {
+      .#{$namespace}-table__cell {
+        border-right: getCssVar('table-border') !important;
```

Comment:
Can we remove  `!mportant`?

We should not leave styles in the component library that cannot be overridden by the user, but conflicts can be avoided with more complex selectors.

Reply:
Done :D